frontend/src/component/pages/Home.scss

```diff
@@ -14,6 +14,32 @@
         margin-right: 20px;
       }
     }
+
+    .create-short-link-btn {
+      display: flex;
+      justify-content: center;
+    }
+  }
+
+  @media only screen and (max-width: 480px) {
```

Comment:
Nit: I found the following example interesting. Maybe we can try something similar for the next refactor ticket:
https://github.com/sass/sass/issues/554#issuecomment-18306483